### PR TITLE
Add bls signature custom account example

### PR DESCRIPTION
**Pull request by jayz22**
Add a custom account example illustrating the use of BLS-signature.

### What

[TODO: Short statement about what is changing.]

### Why

[TODO: Why this change is being made. Include any context required to understand the why.]

### Known limitations

[TODO or N/A]

### CODE REVIEW

**Review by leighmcculloch — COMMENTED** [review]
I didn't review the cryptography. But from the perspective of an example it looks good. @tomerweller Thoughts on who could review this from the perspective of the crypto?

File: bls_signature/Makefile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+default: build
+
+all: test
+
+test: build
+	cargo test
+
+build:
+	soroban contract build
```
Comment (sisuresh) [inline bls_signature/Makefile:9]:
```suggestion
	stellar contract build
```

**Comment by tomerweller** [comment]
I suggest that we publish this as-is with a disclaimer that this is a proof of concept, not audited code

**Comment by leighmcculloch** [comment]
+1. @jayz22 Can you add a significant disclaimer to both a readme file for the directory and the source files, then we'll merge it.

**Comment by jayz22** [comment]
> +1. @jayz22 Can you add a significant disclaimer to both a readme file for the example directory and as a header to the source files, then we'll merge it.

I've added the README file with a clear warning message. The message already existed in the code header and the Makefile. 
Thanks!

**Review by leighmcculloch — APPROVED** [review]

**Comment by leighmcculloch** [comment]
Thanks @jayz22 !

**Pull request merged**